test-cypress/plugins/index.js

```diff
@@ -11,6 +11,8 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+const _ = require("lodash");
```

Comment:
I can't see where the lodash lib comes from? It's not listed in the dependencies in package.json